markdown/events/2021/hackathon-october-2021.md

```diff
@@ -1,16 +1,26 @@
 ---
 title: Hackathon - October 2021
-subtitle: A virtual online hackathon to develop nf-core together.
+subtitle: A virtual online hackathon to develop nf-core together
 type: hackathon
 start_date: "2021-10-27"
 start_time: "10:00 CEST"
 end_date: "2021-10-29"
 end_time: "18:00 CEST"
+import_typeform: true
 ---
 
-Please join us for the October 2021 nf-core hackathon!
-This event is entirely virtual and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:
+Mark your calendars to join us for the October 2021 nf-core hackathon!
 
-More details will come soon, registration will open after summer 2021.
-Please check back here for details and keep an eye out on twitter and on Slack,
-but in the mean time - book the dates in your calendar!
+With the recent release of the new nf-core template for Nextflow's DSL2, in this event we will be going full-steam ahead with conversion of nf-core pipelines from DSL1 to DSL2. The main objectives of this hackathon will be adding nf-core modules and rewriting existing pipelines to the new Nextflow language format.
+
+For Nextflow novices, we will be looking for help in writing Nextflow modules of commonly used bioinformatics tools.
+
+For more experienced developers we hope for contributions to writing subworkflows and pipeline code.
+
+In the spirit of collaboration and knowledge sharing, we plan to pair up both novices and experienced developers in pair programming to help get newcomers up to speed on writing and contributing nf-core modules through a pair-programming session for a limited number of participants. Make sure to sign up by September 27, 2021 to participate in pair-programming!
+
+This event is entirely virtual and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:. Do get in touch with us if you'd like to help us out!
```

Comment:
```suggestion
This event is entirely virtual and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:.
```
This is already taken care of in the form, which is probably also the best way to get in touch with us about it, besides slack. I think we should either mention how they can get in touch with us here or leave it out.